projects/ontimize-web-ngx/src/lib/components/input/o-form-service-component.class.ts

```diff
@@ -119,6 +120,7 @@ export class OFormServiceComponent extends OFormDataComponent {
   protected _formDataSubcribe;
   protected _currentIndex;
   protected dialogService: DialogService;
```

Comment:
Please, check if it is necessary `DialogService` after added `OErrorDialogManager`. If it is no necessary anymore remove it and all references